packages/fondue/src/components/Table/Table.tsx

```diff
@@ -73,7 +78,11 @@ const mapToTableAriaProps = (columns: Column[], rows: Row[], hasSort = false): T
             <TableHeader key="table-header" columns={columns}>
                 {(column) => {
                     const allowsSorting = !!(column.sortable && hasSort);
-                    return <AriaColumn allowsSorting={allowsSorting}>{column.name}</AriaColumn>;
+                    return (
+                        <AriaColumn allowsSorting={allowsSorting} {...{ align: column.align }}>
```

Comment:
is this different than
```suggestion
                        <AriaColumn allowsSorting={allowsSorting} align={column.align}>
```
?

Reply:
Only in terms of types. That component(imported from a library) has a determinate set of props, in this case like `allowSorting`. But if I want to pass custom props to it it only worked via spread

Reply:
interesting, cool TS loophole